src/util/test-helper.tsx

```diff
@@ -108,11 +108,17 @@ export namespace TestHelper {
             ),
             defaultValues: formContext?.defaultValues,
         });
-        if (formContext?.currentValues) {
-            Object.keys(formContext.currentValues).forEach((key) => {
-                methods.setValue(key, formContext.currentValues[key]);
-            });
-        }
+
+        useEffect(() => {
+            if (formContext?.currentValues) {
+                Object.keys(formContext.currentValues).forEach((key) => {
+                    methods.setValue(key, formContext.currentValues[key], {
+                        shouldDirty: true,
+                    });
+                });
+            }
+        }, [formContext?.currentValues, methods]);
+
```

Comment:
revert this change as it has no use and we dont want to cause any false positives